packages/labs/protobufjs/ts_proto_library.bzl

```diff
@@ -175,7 +175,7 @@ a `ts_library` can appear, such as in the `deps[]` of another `ts_library`.
 Example:
 
 ```python
-load("//packages/typescript:index.bzl", "ts_library", "ts_proto_library")
+load("@npm//@bazel/typescript:index.bzl", "ts_library", "ts_proto_library")
```

Comment:
Should we point out that this example assumes you installed dependencies under @npm ?

Reply:
I did wonder if we should do that, however the setup for [managed deps example](https://bazelbuild.github.io/rules_nodejs/install.html#using-bazel-managed-dependencies) shows using the `npm` workspace, so I thought it was reasonable to assume that if the user had followed along with the docs, they'd too have the `npm` workspace. 

I can add the note to those examples if we think it's need however